src/components/FeedbackCreateForm/FeedbackCreateForm.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useCallback, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { Button, Form, FormActions, FormAction, FormTextarea, FormInput, ModalContent } from '@taskany/bricks';
+
+import { errorsProvider } from '../../utils/forms';
+import { createFeedbackSchema, CreateFeedback } from '../../schema/feedback';
+import { ModalEvent, dispatchModalEvent } from '../../utils/dispatchModal';
+
+import { tr } from './FeedbackCreateForm.i18n';
+
+const FeedbackCreateForm: React.FC = () => {
+    const [formBusy, setFormBusy] = useState(false);
+
+    const {
+        register,
+        handleSubmit,
+        formState: { errors, isSubmitted },
+    } = useForm<CreateFeedback>({
+        resolver: zodResolver(createFeedbackSchema),
+    });
+
+    const errorsResolver = errorsProvider(errors, isSubmitted);
+
+    const onPending = useCallback((form: CreateFeedback) => {
+        setFormBusy(true);
+        fetch('/api/feedback', {
+            method: 'POST',
+            body: JSON.stringify({
+                title: form.title,
+                description: form.description,
+                href: window.location.href,
+            }),
```

Comment:
this fetch can fail and you should inform the user about that

Reply:
how?